cli/pkg/cmd/parsing_test.go

```diff
@@ -0,0 +1,140 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright freiheit.com*/
+
+package cmd
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+)
+
+// Used to compare two error message strings, needed because errors.Is(fmt.Errorf(text),fmt.Errorf(text)) == false
+type errMatcher struct {
+	msg string
+}
+
+func (e errMatcher) Error() string {
+	return e.msg
+}
+
+func (e errMatcher) Is(err error) bool {
+	return e.Error() == err.Error()
+}
+
+func ptrStr(s string) *string {
+	return &s
+}
+
+func TestParseArgs(t *testing.T) {
+	type TestCase struct {
+		name             string
+		cmdArgs          string
+		expectedParams   *kuberpultClientParameters
+		expectedOther    []string
+		expectedErrorMsg string
```

Comment:
please compare errors as described in our best practices: https://github.com/freiheit-com/kuberpult/blob/main/README_DEVELOPERS.md#always-use-cmpdiff
```suggestion
		expectedError error
```
This should also make the rather involved `if` statement down below simpler.